pagopa/io-app
# [IODPAY-175] Add IDPay initiative details screen

## Short description
This PR adds the details page for the IDPay initiative.

https://user-images.githubusercontent.com/6160324/234922346-9654208b-7bec-4000-9377-666e8dd44a3b.mp4

## List of changes proposed in this pull request
TBD

## How to test
First, check that with `yarn test:dev` tests solve correctly.
With the `io-dev-server`, navigate to the initiative screen and tap in the icon in the top right section of the header. Check that the data is correctly displayed
